version.txt

```diff
@@ -0,0 +1 @@
+0.6.0
```

Comment:
want to test it with an alpha/beta tag first?

Reply:
Good idea